cet_website_sale/views/website_sale_templates.xml

```diff
@@ -484,14 +486,31 @@
               <input type="radio" name="product_id"
                 t-if="radio_position == 'after'"
                 class="js_product_change radio-after"
-                t-att-id="variant_id.id"
+                t-att-vid="variant_id.id"
                 t-att-checked="'checked' if variant_id.default_variant else None"
                 t-att-value="variant_id.id"
                 t-att-data-lst_price="compute_currency(variant_id.lst_price)"
                 t-att-data-price="variant_id.price"/>
             </div>
           </t>
         </t>
+
+        <!-- This select tag is just a wrapper that uses the input above -->
+        <select class="oe_cet_product_variant_select js_product_change_select">
+          <t t-foreach="product.product_variant_ids" t-as="variant_id">
+            <t t-if="not restrict_product or (restrict_product and variant_id in allowed_products)">
+              <option t-if="variant_id.variant_sale_ok"
+                  t-att-checked="'checked' if variant_id.default_variant else None"
```

Comment:
`checked` is working ? I thought it should be `t-att-selected="'selected' if variant_id.default_variant else None"` for an `<option>` : https://www.w3schools.com/tags/att_option_selected.asp

Reply:
I fact, `selected="selected"` is same as `selected`. Here with `t-att-` if expression is `None` odoo will omit to add `selected` attribute.